src/components/room-view.module.scss

```diff
@@ -0,0 +1,22 @@
+.container {
+    width: 100%;
+    height: 70px;
+    box-sizing: border-box;
+    display: flex;
+    padding: 5px 5px 5px 15px;
+  }
+  .name{
```

Comment:
do not regret empty lines between different code blocks, you have the unlimited, and with them, you could really make code readability better